report/2.Analysis/ArduinoLanguagesCtrl.tex

```diff
@@ -1,70 +1,130 @@
 \section{Existing languages}
-Before doing any work on HCL, the project group decided to research how many high-level languages already exist for the Arduino platform, and what distinguishes them from each other.
-Surprisingly, despite the popularity of the platform, there are not many options available.
-Some available languages are:
+Before doing any work on HCL, the project group has decided to research how many high-level languages already exist for the Arduino platform, and what distinguishes them from each other.
+This was done in part to find out what kind of competition there may be for HCL, and in part to gain inspiration from the existing languages.
+There is a variety of languages to choose from when programming for the Arduino.
+Some of these are:
```

Comment:
Don't know if you here should say something along the lines of: "The project group looked at the following". I think Some of these is a little too "talk language"